### ISSUE TITLE
Avoid composer lock diff action if composer.lock hasn't changed

### ISSUE THREAD

**Issue by beto-aveiga** [Lullabot/drainpipe#677]
Composer lock diff action takes about ~3 minutes on a minimal site.
There is no need to run this workflow if there are no composer changes, or we could avoid running it for 3 minutes (depending on the site, this may vary).
We could skip it in the mentioned scenario and use less than 1 minute.

**Comment by deviantintegral** [comment]
Good call. I think we can do this with a path filter in GitHub at least: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

**Comment by alexanderpatriciop** [comment]
@deviantintegral Good suggestion, I tested it on a test project and it works as desired.
